src/main/java/com/ramcel/cinema/reservation/functionalities/ticket/TicketType.java

```diff
@@ -0,0 +1,18 @@
+package com.ramcel.cinema.reservation.functionalities.ticket;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@AllArgsConstructor
+@Getter
+public enum TicketType {
+    ADULT(BigDecimal.valueOf(25)),
+    STUDENT(BigDecimal.valueOf(18)),
+    CHILD(BigDecimal.valueOf(12.5));
```

Comment:
Fajnie, że użyłeś `BigDecimal` - a dlaczego nie np. `Double`? :slightly_smiling_face: 

Czy kwoty biletów moglibyśmy umieścić w innym miejscu, by nie musieć przebudowywać aplikacji gdy przyjdą np. podwyżki inflacyjne?

Reply:
BigDecimal ma duży overhead, ale jest precyzyjne - za dużo obliczeń nie wykonujemy, więc chyba lepiej dokładnie policzyć ile chcemy dostać pieniędzy :). 

Wartości biletów możnaby np wczytać z zewnętrznego pliku springa i konfigurować za pomocą \@Value, można zdefiniować Bean w konfiguracji springa. Są też pewnie jakieś metody na pobieranie tego z bazy danych, ale nie bawiłem się w to nigdy.